drivers/sensor/pmw3360/pmw3360.c

```diff
@@ -441,7 +441,7 @@ static int update_cpi(const struct device *dev, uint32_t cpi)
 	/* Convert CPI to register value */
 	uint8_t value = (cpi / 100) - 1;
 
-	LOG_INF("Setting CPI to %u (reg value 0x%x)", cpi, value);
+	LOG_DBG("Setting CPI to %u (reg value 0x%x)", cpi, value);
```

Comment:
Shouldn't we rather decrease the logger configuration for this specific module in the nRF Desktop application?

Reply:
I think that it may be good to reduce verbosity of motion sensor related logs by default (we still inform if something went wrong with warning/error logs). Application may turn on the debug logs to get more information about the sensor if needed (e.g. during debugging).